src/ThriftSQL/ResultRow.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace ThriftSQL;
+
+class ResultRow implements \ArrayAccess
+{
+  private $_row;
+  private $_schema;
+
+  public function __construct($schema, $row)
+  {
+    $this->_row = $row;
+    $this->_schema = $schema;
+  }
+
+  public function schemaArray()
+  {
+    $r = [];
+    foreach ($this->_schema as $k=>$i) {
+      $r[$k] = $this->_row[$i];
+    }
+    return $r;
+  }
```

Comment:
Is this function needed if `\ThriftSQL\ResultRow` already implements `\ArrayAccess`? Shouldn't the consumer of the API be able to access the row data by key in which case seems like array casting would also work to accomplish the same thing if the consumer wants an actual array rather then an object that implements `\ArrayAccess`.

Reply:
In my case, I need to translate the `ResultRow`'s keys to another array, so I think it's better to expose the assoc array for the consumer.